doc/nrf/ug_multi_image.rst

```diff
@@ -24,12 +42,143 @@ MCUboot bootloader
+   When programming applications to the nRF5340 PDK, they must be divided into at least two images, one for each core.
+   See :ref:`ug_nrf5340` for more information.
+
+   .. important::
+      Currently, the two images must be built and programmed separately.
+      Multi-image builds are not yet supported for nRF5340.
+
+
+Default configuration
+*********************
+
+The |NCS| samples are set up to build all related images as one solution, starting from the parent image.
+This is referred to as *multi-image build*.
+
+When building the parent image, you can configure how the child image should be handled:
+
+* Build the child image from source and include it with the parent image.
+  This is the default setting.
+* Use a prebuilt HEX file of the child image and include it with the parent image.
+* Ignore the child image.
+
+When building the child image from source or using a prebuilt HEX file, the build system merges the HEX files of the parent and child image together so that they can easily be programmed in one single step.
+This means that you might enable and integrate an additional image without even realizing it, just by using the default configuration.
+
+To change the default configuration and configure how a child image is handled, locate the BUILD_STRATEGY configuration options for the respective child image in the parent image configuration.
+For example, to use a prebuilt HEX file of the :ref:`secure_partition_manager` instead of building it, select :option:`CONFIG_SPM_BUILD_STRATEGY_USE_HEX_FILE` instead of the default :option:`CONFIG_SPM_BUILD_STRATEGY_FROM_SOURCE`, and specify the HEX file in :option:`CONFIG_SPM_HEX_FILE`.
+To ignore an MCUboot child image, select :option:`CONFIG_MCUBOOT_BUILD_STRATEGY_SKIP_BUILD` instead of :option:`CONFIG_MCUBOOT_BUILD_STRATEGY_FROM_SOURCE`.
+
+
+Defining and enabling a child image
+***********************************
+
+You can enable existing child images in the |NCS| by enabling the respective modules in the parent image and selecting the desired build strategy.
+To turn an application that you have implemented into a child image that can be included in a parent image, you must update the build scripts to make it possible to enable the child image and add the required configuration options.
+You should also know how image-specific variables are disambiguated and what targets of the child images are available.
+
+Updating the build scripts
+==========================
+
+To make it possible to enable a child image from a parent image, you must include the child image in the build script.
+
+This code should be put in place in the cmake tree that is conditional on a configuration option for having the parent image use the child image.
+In the |NCS|, the code is included in the :file:`CMakeLists.txt` file for the samples, and in the MCUboot repository.
+
+See the following example code:
+
+.. code-block:: cmake
```

Comment:
I added the other two examples here since I feel they are more straight-forward - does that make sense or is it confusing?

Reply:
That's ok.